templates/v2/professional/_action_dropdown.html.twig

```diff
@@ -17,11 +21,19 @@
 >
     <i class="fas fa-ellipsis-h"></i>
 </button>
-<div class="position-relative d-none" data-dropdown-menu-target="menu">
+<div class="position-relative d-none" data-dropdown-menu-target="menu"
+        {{ stimulus_controller('alert', {
+            'confirmButtonText': 'main.accepter'|trans,
+            'cancelButtonText': 'main.annuler'|trans,
+        }) }}>
     <ul class="action-dropdown shadow border-1 position-absolute z-1 bg-white list-group font-size-1 mt-2 end-0" style="width: 240px">
```

Comment:
I think we should'nt use legacy translation keys in v2, even if it means having duplicates. It will make it much simplier to delete legacy translations keys later.